I am a PhD candidate in bioinformatics, a first class honours graduate in computer engineering, a computer and electronics enthusiast, an open-source software developer and advocate, an avid traveller, a hiker, a singer, and an experimental cook. I have fifteen years of industry and academic experience developing software in C++ and a variety of scripting languages, including nine years developing high-performance distributed and parallel algorithms for genome sequence assembly, and six years developing embedded and real-time control systems. I have extensive experience developing portable software for POSIX systems such as Linux and macOS. When developing performance-critical software, I understand the hardware from the transistor up.

## Education

#### 2012–2017 PhD Plant breeding, Genetics & Genomics — University of Missouri Columbia, USA

•	Received Dr. Marcus Zuber endowment fellowship
•	Presented research at Maize genetics conference, Plant Animal Genetics conference and The Tri-Societies meeting (ASA, CSSA, SSSA)       International meetings
•	Recipient of 2017 Research in Plant Biology Award and Life Sciences Week Research Recognition award

#### 2008–2012 BSc Agricultural Sciences — Truman State University, USA

•	Received President’s Honorary Scholarship and Dr. Max E. Bell Scholarship
•	Presented research at student research conferences at Truman, Adair County extension center and annual Maize genetics conference

## Work Experience

#### 2018–present Computational biologist and Quantitative Geneticist — CORNELL UNIVERSITY, Geneva, NY
•	Analyzed of large-scale genotyping data (from Genotyping-by-Sequencing, Amplicon Sequencing, Illumina resequencing, and/or PacBio de     novo sequencing) such as SNP and Haplotype calling. 
•	Developed genetic markers for traits like powdery mildew disease resistance and fruit quality, to assist breeders with marker assisted   selection; conducted statistical analysis of large-scale data sets from DNA sequencing and trait data from related and unrelated vines   for genetic mapping in Lep-MAP3, QTL mapping in R, genome wide association studies in TASSEL/FarmCPU/GAPIT.
•	Managed grapevine breeding database www.VitisBase.org , and led setting up trials and phenotype data collection on Field Book android   application.

#### 2017 May–2017 Dec Quantitative Geneticist — UNIVERSITY of ILLINOIS, Carl R. Woese Institute for Genomic Biology	                  Urbana–Champaign, IL, USA

Developed the firmware for Pathway's product line of small, low-cost lighting control devices. These small devices, which use Atmel's AVR and ARM microcontrollers, convert between a variety of lighting-control protocols. Modernized the lighting control industry in developing a device to adapt legacy lighting control equipment to Ethernet. This industry-changing product, the Pathport, won an award for best new product at the industry trade show, Lighting Dimensions International.

#### 2002 May—Co-op work experience—Vortek Industries Ltd.

Worked on the rapid thermal processing tool used in the manufacturing
of semiconductor devices. Developed the temperature sensor and lamp
power control system using the QNX real-time operating system.

## Volunteer Experience

#### 2002–present—Open-source developer

Maintain open-source bioinformatics software packages for Homebrew and Linuxbrew. Lead developer of Linuxbrew, the Homebrew package manager for Linux.

## Selected Publications

A selection of 12 of 30 peer-reviewed publications, h-index of 21, and over 7,000 citations

#### Tigmint: Correcting assembly errors using linked reads from large molecules
Jackman SD, Coombe L, Chu J, *et al.*
(2018)
*BMC Bioinformatics*

#### ABySS 2.0: Resource-efficient assembly of large genomes using a Bloom filter
**Jackman SD**, Vandervalk BP, Mohamadi H, Chu J, Yeo S, Hammond SA, Jahesh G, Khan H, Coombe L, Warren RL, et al.
(2017)
*Genome Research*

#### Organellar Genomes of White Spruce (*Picea glauca*): Assembly and Annotation
**Jackman SD**, Warren RL, Gibb EA, Vandervalk BP, Mohamadi H, Chu J, Raymond A, Pleasance S, Coope R, Wildung MR, et al.
(2015)
*Genome Biology and Evolution*

#### Improved white spruce (*Picea glauca*) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism
Warren RL, Keeling CI, Yuen MMS, Raymond A, Taylor GA, Vandervalk BP, Mohamadi H, Paulino D, Chiu R, **Jackman SD**, et al.
(2015)
*The Plant Journal*

#### UniqTag: Content-Derived Unique and Stable Identifiers for Gene Annotation
**Jackman SD**, Bohlmann J, Birol İ.
(2015)
*PLOS ONE*

#### Assembling the 20 Gb white spruce (*Picea glauca*) genome from WGSS data
I Birol, A Raymond, **SD Jackman**, et al.
(2013)
_Bioinformatics_

#### Draft genome of the mountain pine beetle, *Dendroctonus ponderosae* Hopkins
CI Keeling, MM Yuen, NY Liao, TR Docking, SK Chan, GA Taylor, DL Palmquist, **SD Jackman**, et al.
(2013)
_Genome Biology_

#### De novo assembly and analysis of RNA-seq data
G Robertson, J Schein, R Chiu, R Corbett, M Field, **SD Jackman** et al.
(2010)
_Nature Methods_

#### Assembling genomes using short-read sequencing technology
**SD Jackman** and I Birol
(2010)
_Genome Biology_

#### De novo transcriptome assembly with ABySS
I Birol, **SD Jackman**, CB Nielsen et al.
(2009)
_Bioinformatics_

#### ABySS-Explorer: visualizing genome sequence assemblies
CB Nielsen, **SD Jackman**, I Birol et al.
(2009)
_IEEE Transactions on Visualization and Computer Graphics_

#### ABySS: a parallel assembler for short read sequence data
JT Simpson, K Wong, **SD Jackman** et al.
(2009)
_Genome Research_

## Knowledge

#### Bioinformatics

Techniques for analysis of whole-genome shotgun sequencing and RNA-seq, including *de novo* sequence assembly, sequence alignment, variant calling, genome annotation, and visualization and inspection of genome assemblies, variants, and rearrangements.

#### Programming languages

C++, C, R, Python, Ruby, shell, and assembly.

#### Parallel processing

OpenMP, MPI, and POSIX threads.

#### Operating systems

macOS, Debian, Ubuntu, and other GNU/Linux distributions.

## Awards

#### University of British Columbia

* 2013-2018 Natural Sciences and Engineering Research Council of Canada (NSERC)
  Canada Graduate Scholarship (CGS-M and CGS-D)
* 2012–2013 CIHR/MSFHR Strategic Training Program in Bioinformatics

#### Simon Fraser University

* 1999–2004 Open Undergraduate Scholarship
* 1998–1999 Tadeusz Specht Memorial Entrance Scholarship in Science

